cmd/root.go

```diff
@@ -41,6 +41,18 @@ var (
 			if rootCmdOpts.debug {
 				logrus.SetLevel(logrus.TraceLevel)
 			}
+			var stat unix.Statfs_t
+
+			unix.Statfs(rootCmdOpts.dir, &stat)
+
+			avail := stat.Bavail * uint64(stat.Bsize)
+
+			var disk_threshold uint64 = 10000000
```

Comment:
Could we have this configurable, as an argument. We should have also the query interval as an argument.